persistence/src/main/scala/za/co/absa/spline/persistence/model/entities.scala

```diff
@@ -63,5 +63,12 @@ case class Progress(
   timestamp: Long,
   error: Option[Any],
   extra: Map[String, Any],
-  override val _key: String
+  override val _key: String,
+  // the rest of the parameters is copied from other entities - for performance reasons
```

Comment:
I wonder if it wouldn't make sense to nest these fields into another one called say `execPlanDetails`? This would make the above comment unnecessary, and it would also look more clear from the places of usage.